### PR TITLE
[Develop] new component Image

### PR DESCRIPTION
**Pull request by naimdanialjohar**
# Description

new component Image. [Jira ticket](https://kfit-asia.atlassian.net/browse/FAVE-17114)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)
image
image
image
image

### CODE REVIEW

File: src/Image/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Image as AntImage, ImageProps as AntImageProps } from 'antd';
+import './style.css';
+
+export type ImageProps = AntImageProps;
+export const fallbackIcon =
```
Comment (AladdinNullptr) [inline src/Image/index.tsx:6]: When using svg file in code, you can put them under `/assets/icons` as you did previously. The base64 format is needed when we render icons in css

File: src/Image/style.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@import 'antd/lib/image/style/index.css';
+@import 'antd/lib/style/default.css';
+
```
Comment (AladdinNullptr) [inline src/Image/style.css:3]: Import order is important. Always import styles in the same order as the library. If you see `\antd\lib\image\style\css.js`, `default` style comes before `image` style

Comment (naimdanialjohar) [reply src/Image/style.css:3]: ahh okay noted. will do the necessary changes

File: src/Image/index.tsx (hunk 1 above)
Comment (naimdanialjohar) [reply src/Image/index.tsx:6]: the `fallbackIcon` prop only accepts type `string` to be passed as the value, like a URL for example.

**Review by AladdinNullptr — APPROVED** [review]

**Pull request merged**